draft-tus-resumable-uploads-protocol.md

```diff
@@ -138,7 +158,11 @@ Client                                      Server
 ~~~
 {: #fig-upload-cancellation-procedure title="Upload Cancellation Procedure"}
 
-In the above example, the client attempted to upload the entire file in a single request, indicated by omitting the `Upload-Incomplete` header. For advanced use cases, the client is allowed to upload incomplete chunks of a file to the server sequentially. One example is the uploading of a streaming data source which is not yet completely read yet:
+## Example 2: File of unknown size
+
+In some cases a client does no know the final size of a resource before it initiates an upload. For instance, a file that is contains data from a stream source.
```

Comment:
Extra "is"? "a file is contains data"

Reply:
Side note: Not really sure that this part of the procedure is due to not knowing the final upload size which this sentence seems to indicate. I would rather use this to bypass proxy max size etc.

Reply:
Good comment. I think the current text on "advanced use cases" is not descriptive enough to be useful.

I don't particularly like the use of chunks in this section of the spec because it might be inferred as chunked encoding. If a client uploads the "whole file" even using chunked encoding, proxies can still implement a content size limit.

I think instead what you're saying is that one use case is: in order to overcome proxy HTTP message content size limits to, clients break a large resource into smaller parts and transfer it over a series of messages from the get go (if they have prior knowledge the action would be supported). I will attempt to rephrase the example to accommodate that.